best-practices/tagging/index.html

```diff
@@ -77,9 +89,9 @@ <h3>eBraille tagging</h3>
 		
 			<section id="boxes">
 				<h4>Boxed text</h4>
-
+				<p>Boxes would be used any time that text needs to be enclosed in a full lines and/or columns of braille characters. The specific braille characters and where they appear will be determined by CSS. As an example, you might have text that is preceded and followed by a full line of the same braille character but you could also have variations or full columns of braille characters flanking the text on either side.</p>
```

Comment:
```suggestion
				<p>Boxes are used any time that text needs to be enclosed in full lines and/or columns of braille characters. The specific braille characters and where they appear will be determined by CSS. As an example, you might have text that is preceded and followed by a full line of the same braille character but you could also have variations or full columns of braille characters flanking the text on either side.</p>
```

Reply:
Fixed through responding to James' comments.